uxarray/core/dataarray.py

```diff
@@ -97,3 +96,113 @@ def uxgrid(self):
     @uxgrid.setter
     def uxgrid(self, ugrid_obj):
         self._uxgrid = ugrid_obj
+
+    def to_geodataframe(self,
+                        override_geometry=False,
+                        cache_geometry=True,
```

Comment:
I read your latest comment on the naming of these, and it makes sense to me; however, I think the description of these vars below is well written and self-explanatory, so I still think naming these as `cache` and `override` just like you did in `grid.to_geodataframe()` and `grid.to_polycollection` should be fine.